sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/src/EventGridExtensionConfigProvider.cs

```diff
@@ -34,6 +35,11 @@ internal class EventGridExtensionConfigProvider : IExtensionConfigProvider,
         private readonly ILoggerFactory _loggerFactory;
         private readonly Func<EventGridAttribute, IAsyncCollector<object>> _converter;
         private readonly HttpRequestProcessor _httpRequestProcessor;
+        private readonly DiagnosticScopeFactory _diagnosticScopeFactory;
+
+        // ApplicationInsights SDK listens to all Azure SDK sources that look like 'Azure.*'
+        private const string DiagnosticScopeNamespace = "Azure.WebJobs.Extensions.EventGrid";
```

Comment:
Having to use a different namespace is a bit unfortunate. But I don't see a way around it. Should we use `Azure.Messaging.EventGrid` so it aligns with other EG spans?